### PR TITLE
[#2979] Inline storage support importing data from external data sources

### PR DESCRIPTION
**Pull request by chuntaojun**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

#2979 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/test/java/com/alibaba/nacos/client/NamingTest.java [hunk 1]
```diff
@@ -61,7 +62,8 @@ public void testServiceList() throws Exception {
 
         System.out.println(list);
 
-        ThreadUtils.sleep(60_000L);
+        CountDownLatch latch = new CountDownLatch(1);
+        latch.await();
```
Comment (KomachiSion) [inline client/src/test/java/com/alibaba/nacos/client/NamingTest.java:66]: When the latch will count down?

File: config/src/main/java/com/alibaba/nacos/config/server/filter/CurcuitFilter.java [hunk 2]
```diff
@@ -53,9 +51,11 @@
  *
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
-@SuppressWarnings("all")
 public class CurcuitFilter implements Filter {
 
+	private volatile boolean isDowngrading = false;
+	private volatile boolean isOpenService = false;
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/filter/CurcuitFilter.java:57]: blank line between varaible.

File: core/src/main/java/com/alibaba/nacos/core/utils/DiskUtils.java [hunk 3]
```diff
@@ -87,6 +90,81 @@ public static void touch(File file) throws IOException {
...
+     *          if the array contains an attribute that cannot be set atomically
+     *          when creating the directory
+     * @throws  IOException
+     *          if an I/O error occurs or {@code dir} does not exist
+     * @throws  SecurityException
+     *          In the case of the default provider, and a security manager is
+     *          installed, the {@link SecurityManager#checkWrite(String) checkWrite}
+     *          method is invoked to check write access to the file.
+     */
+    public static File createTmpFile(String dir, String prefix, String suffix) throws IOException {
```
Comment (KomachiSion) [inline core/src/main/java/com/alibaba/nacos/core/utils/DiskUtils.java:131]: Is the two method copied?

Comment (chuntaojun) [reply core/src/main/java/com/alibaba/nacos/core/utils/DiskUtils.java:131]: No, just integrate the method into DiskUtils

**Pull request merged**